iam/policy-templates/dss-scalability-test-lambda.json

```diff
@@ -0,0 +1,73 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "logs:CreateLogGroup",
+        "logs:CreateLogStream",
+        "logs:DescribeLogGroups",
+        "logs:DescribeLogStreams",
+        "logs:PutLogEvents"
+      ],
+      "Resource": "arn:aws:logs:*:*:*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "s3:List*",
+        "s3:Get*",
+        "s3:PutObject*",
+        "s3:DeleteObject*"
```

Comment:
This is not an appropriate permission to give to this daemon. Let's review this offline.

Reply:
Delete was no longer needed, removed